server/src/main/java/org/opensearch/rest/action/cat/RestHealthAction.java

```diff
@@ -88,7 +88,12 @@ protected Table getTableWithHeader(final RestRequest request) {
         t.addCell("status", "alias:st;desc:health status");
         t.addCell("node.total", "alias:nt,nodeTotal;text-align:right;desc:total number of nodes");
         t.addCell("node.data", "alias:nd,nodeData;text-align:right;desc:number of nodes that can store data");
-        t.addCell("discovered_master", "alias:dm;text-align:right;desc:discovered master");
```

Comment:
With the Cluster Health API - #2437 - we chose to _add_ a "cluster_manager" field alongside the "master" one, but here we're _replacing_ "master" with "cluster_manager". Why the difference in approach?

Reply:
D'oh, should've read the PR description more carefully 😅 

To answer my own question - #2437 is adding a new field in the response, so we cannot replace it in-line. In this PR, the change affects the human-readable table header in the output, which we hardly expect to be parsed by code.

Feel free to ignore my question above 😄

Reply:
Hi @kartg, I really appreciate your careful attitude, to put the explanation in detail.
It's also my negligence that I didn't explain more in this PR description, after add more description in anther PR with the same concern: https://github.com/opensearch-project/OpenSearch/pull/2441#issuecomment-1068547454 .
I will update in this PR description to encourage users who wants to use the same API request to parse the response of CAT Health in 2.x nodes, to filter output by specifying table header `GET _cat/health?v&h=discovered_master`